packages/one-app-bundler/bin/bundle-module.js

```diff
@@ -13,22 +13,40 @@
  * under the License.
  */
 
-
-const webpack = require('webpack');
-const path = require('path');
-const fs = require('fs');
 const localeBundler = require('@americanexpress/one-app-locale-bundler');
 
 const clientConfig = require('../webpack/module/webpack.client');
 const serverConfig = require('../webpack/module/webpack.server');
-const getWebpackCallback = require('./webpackCallback');
+const buildWebpack = require('../utils/buildWebpack');
+const time = require('../utils/time');
 const { watch } = require('../utils/getCliOptions')();
 
-const modernClientConfig = clientConfig('modern');
-const legacyClientConfig = clientConfig('legacy');
+(async function bundleModule() {
+  await (
+    time(() => localeBundler(watch))
+      .then((timeInMs) => {
+        console.log(timeInMs);
+      })
+      .catch((error) => {
+        console.error(error);
+      })
+  );
+
+  await (
+    time(() => {
+      const configs = [
+        ['node', serverConfig],
+        ['browser', clientConfig('modern')],
+        ['legacyBrowser', clientConfig('legacy')],
+      ].map(([name, config]) => ({
+        ...config,
+        name,
+      }));
 
-fs.writeFileSync(path.join(process.cwd(), 'bundle.integrity.manifest.json'), JSON.stringify({}));
-localeBundler(watch);
-webpack(serverConfig, getWebpackCallback('node', true));
-webpack(modernClientConfig, getWebpackCallback('browser', true));
-webpack(legacyClientConfig, getWebpackCallback('legacyBrowser', true));
+      return buildWebpack(configs, { watch });
+    })
+      .then((timeInMs) => {
+        console.log(timeInMs);
```

Comment:
This would give the team it took for all 3 to be built right? Would it be beneficial to state how long all 3 took individually?

Reply:
Possibly... Do you think that level of detail is needed? I think having the total build time for all three webpack builds is fine.

Reply:
We can use the `stats` to get the time taken to build in milliseconds if we wanted:
https://webpack.js.org/api/stats/